pkg/apps/ntp.go

```diff
@@ -0,0 +1,40 @@
+package apps
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/flanksource/konfigadm/pkg/types"
+)
+
+// NTP represents a phase which gives information
+// on how to install NTP on the system
+var NTP Phase = ntp{}
+
+type ntp struct{}
+
+func (n ntp) ApplyPhase(sys *Config, ctx *SystemContext) ([]Command, Filesystem, error) {
```

Comment:
Can we add a Verify function that ensure that the ntp service is running and healthy and that /etc/ntp.conf contains the expected content ?

Reply:
Sure 👍  I'll make the changes and comment here